client/verta/verta/_deployment/endpoint.py

```diff
@@ -407,6 +407,26 @@ def get_access_token(self):
             return None
         return tokens[0]['creator_request']['value']
 
+    def create_access_token(self, token):
```

Comment:
Let's also add a typecheck for token: is `six.string_types`

Reply:
Fixed.